custom_components/zwave_mqtt/entity.py

```diff
@@ -202,12 +203,12 @@ def device_state_attributes(self):
     def name(self):
         """Return the name of the entity."""
         node = self.values.primary.node
-        return f"{node.node_manufacturer_name} {node.node_product_name}: {self.values.primary.label}"
+        return f"{create_device_name(node)}: {self.values.primary.label}"
 
     @property
     def unique_id(self):
         """Return the unique_id of the entity."""
-        return self.values.unique_id
+        return f"{DOMAIN}.{self.platform.domain}.{self.values.values_id}"
```

Comment:
The entity registry is aware of both integration domain and platform domain. We can remove this.

Reply:
Sure, I tried that before and I was getting unique_id errors without the platform.domain in the unique_id. Remember that we have sensors based on same values set in 2 different platforms (binary_sensor and sensor)

Reply:
Unique ids only have to be unique per platform and integration. Only if there is more than one type of entity within the same platform for a device, do we need to add the entity type to the unique id. This is common in binary and sensor platforms may spawn multiple entities for the same device.

Reply:
You are right, I got on the wrong track with this because of those "platform does not generate unique id's" error but that is related to the reloading of the config entry and has has nothing to do with this PR. Reverted to the unique ID of the primary OZW Value again,